assets/scripts/segments/__tests__/__snapshots__/Segment.test.js.snap

```diff
@@ -0,0 +1,351 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Segment renders active segment correctly 1`] = `
+<div
+  className="segment hover show-drag-handles"
+  data-width={1}
+  onMouseEnter={[Function]}
+  onMouseLeave={[Function]}
+  style={
+    Object {
+      "undefined": "translateX(undefinedpx)",
```

Comment:
There's various instances where `undefined` shows up in the snapshot, instead of real values (or real-looking dummy values). Is this OK?